defaults/main.yml

```diff
@@ -95,3 +95,14 @@ odoo_role_enable_sentry: false
 # Overwrites existing translation terms on updating a module or importing a CSV or a PO file using the option --i18n-overwrite in the module update.
 # https://www.odoo.com/documentation/12.0/developer/reference/cmdline.html#cmdoption-odoo-bin-i18n-overwrite
 odoo_role_i18n_overwrite: false
+logrotate_odoo:
```

Comment:
Should this variable start with `odoo_role` as the other variables? Like `odoo_role_logrotate_odoo`, for instance.